packages/server-wallet/src/wallet/wallet.ts

```diff
@@ -464,6 +464,14 @@ export class Wallet extends EventEmitter<WalletEvents> {
     return channelResults;
   }
 
+  /**
+   * Registers a peer with the message service so messages can be sent to the peer.
+   * @param peerUrl The endpoint for the peer.
+   */
+  public async registerPeerMessageService(peerUrl: string): Promise<void> {
+    await this.messageService.registerPeer(peerUrl);
```

Comment:
I'm not sure if this `await` matters? If the caller doesn't `await` `registerPeerMessageService` then I don't think this `await` would block it, since the caller is not waiting for promise at all.

I've played around in the typescript playground [here](https://www.typescriptlang.org/play?module=1#code/KYDwDg9gTgLgBAQwM4E8B2BjOAzArpmASwjTgHcFCYBBAISoAoBKALgAUoIBbQpYAHgBuEQgBMAfAG8AUHDlwowGLiik0wMnA7dewBoqQQANoOABecXxgAVQl2ARcMBswuSMJQ0eAA6IxABzBgByCio6KjgAERJgYKYAbgVgL1NmBIBfABo4AEYABkKmROkM6WlQSFhEVEwcfAwiEjgYFJgAMQamtGYWLU4ePiERCRl5OA80L19-IODaYGxoYHJKGnoYeITpBDD4VfCN9Nl5Sem-QJDqbFaoA-WqLfKy8ukz4xnL+cXllrbOgjENBPP5IDpdIHHd7eC5zBA3YB3VpggGNIFPaRAA) trying to convince myself that is the case.